contents/stats_distributions.md

```diff
@@ -1,45 +1,93 @@
-2. **Additivity**: For two mutually exclusive events $A$ and $B$ (cannot occur at the same time): $P(A) = 1 - P(B)$ and $P(B) = 1 - P(A)$.
-3. **Normalization**: The probability of all possible events $A_1, A_2, \dots$ must add up to 1: $\sum_{n \in \mathbb{N}} P(A_n) = 1$; or, in the case of a continuous variable, integrate to 1: $\int^\infty_{-\infty} f(x) dx = 1$.
+Okay, so over 11 rows, the ball fell `jl x` times to the right.
+What if we do the same thing a few more times?
 
-Now we are ready to talk about distributions.
-Simply put: a **distribution is just a collection of scores (counts) in a variable (observations)**.
-A **probability distribution is defined by a function which maps real numbers to a probability**:
+```jl
+s = """
+    n_repeats = 800
+    X = [count(rand(Bool, n_rows)) for i in 1:n_repeats]
+    X[1:10]
+    """
+sco(s)
+```
 
-$$ f(X): X \to \mathbb{R} \in [0, 1], $$ {#eq:distribution}
+Apparently, the ball doesn't always fall `jl x` times to the right.
+We can create a scatter plot for these numbers:
 
-where $\mathbb{R}$ is the set of real numbers and $[0, 1] \in \mathbb{R}$ is the closed interval of real numbers between 0 and 1.
-In other words, distributions are a mapping between values and their respective probabilities.
-We generally denote that a random variable $X$ follows a specific distribution by the following notation:
+```jl
+s = """
+    CairoMakie.activate!() # hide
+    figure = (; resolution=(600, 400))
+    xlabel = "number of times fallen to the right"
+    axis = (; xlabel, ylabel="ball")
+    scatter(X, 1:n_repeats; figure, axis)
+    label = "first_distribution" # hide
+    link_attributes = "width=80%" # hide
+    caption = "Scatter plot for balls falling on pins." # hide
+    Options(current_figure(); filename=label, caption, label, link_attributes) # hide
+    """
+sco(s)
+```
 
-$$ X \sim \operatorname{dist}(\theta_1, \theta_2, \dots), $$ {#eq:distribution_notation}
+To answer our question about how many times the ball will fall to the right, we can apply a trick.
+We can't answer the question exactly, but we can make a guess based on how many balls have fallen in each region.
+For example, when looking at @fig:first_distribution, we can conclude that it is very unlikely that a ball will fall 0 times to the right or 100 times to the right.
+It is more likely that a ball will fall roughly half of the time to the left and half of the time to the right.
+In other words, that the ball will fall 50 times to the right.
+But, it also clearly isn't the case that the ball always falls 50 times to the right.
+A question that we can answer is: how many times could we expect the ball to fall 40 times to the right? What about 60 times?
+To answer that, we can divide the range $[0, 100]$ into bins and count how many balls have fallen in that range.
+This is called a **hist**ogram:
```

Comment:
We might need to back-reference this at the stats-vis sections regarding histograms.